src/components/ui/AppColorPicker.vue

```diff
@@ -74,8 +74,29 @@
           />
         </v-layout>
 
+        <div class="text-right mt-4 mr-1">
+          <v-btn-toggle
+            v-model="valueRange"
+            color="primary"
+            mandatory
+          >
+            <v-btn
+              x-small
+              value="absolute"
+            >
+              0..255
+            </v-btn>
+            <v-btn
+              x-small
+              value="percentage"
+            >
+              0..1
+            </v-btn>
+          </v-btn-toggle>
```

Comment:
Works fine, but I wonder if we should make it match the style used around the rest of the app?

![image](https://github.com/fluidd-core/fluidd/assets/85504/cd1e94e9-059e-4fac-a23c-2adb65ad4e8d)

For this I replaced `v-btn` with `app-btn` and removed the `color="primary"` from the top

```suggestion
          <v-btn-toggle
            v-model="valueRange"
            mandatory
          >
            <app-btn
              x-small
              value="absolute"
            >
              0..255
            </app-btn>
            <app-btn
              x-small
              value="percentage"
            >
              0..1
            </app-btn>
          </v-btn-toggle>
```

Reply:
yep, let's keep it consistent! thanks